Riot/Modules/ServiceTerms/Modal/ServiceTermsModalCoordinatorBridgePresenter.swift

```diff
@@ -98,13 +100,36 @@ extension ServiceTermsModalCoordinatorBridgePresenter: ServiceTermsModalCoordina
 
     func serviceTermsModalCoordinatorDidAccept(_ coordinator: ServiceTermsModalCoordinatorType) {
         self.delegate?.serviceTermsModalCoordinatorBridgePresenterDelegateDidAccept(self)
+        
+        if serviceType == MXServiceTypeIdentityService {
```

Comment:
It's better if we put analytics inside Coordinator because BridgePresenters are only built for ObjC compatibility when we will remove the presenter we will loose the analytics.